### PR TITLE
Fill community index page

### PR DESCRIPTION
**Pull request by razvand**
Add general information on the Unikraft open source community, resources and development model.

### CODE REVIEW

File: content/en/community/_index.md [hunk 1]
```diff
@@ -4,10 +4,19 @@ date: 2020-01-11T14:09:21+09:00
...
 weight: 100
 collapsible: true
-enableToc: false
 ---
 
-## Unikraft Open-Source Community
+## Unikraft Open Source Community
 
-The Unikraft Open-Source community, its users, contributors, and the culture
+Unikraft is developed in the spirit of open source: public discussions on [Discord](https://bit.ly/UnikraftDiscord), open source licensing model using [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause), public development and management on [GitHub](https://github.com/unikraft/).
```
Comment (nderjung) [inline content/en/community/_index.md:11]: Maybe link to the licensing page?

File: content/en/community/_index.md [hunk 2]
```diff
@@ -4,10 +4,19 @@ date: 2020-01-11T14:09:21+09:00
...
 
+It does so in the context of a vibrant community consisting of highly skilled software engineers, researchers, teachers, students and hobbysts.
+Periodic meetings, hackathons and a consistent presence in open source events are central to the well functioning of the community.
+
+We use a [guidelines for development and maintenance](docs/contributing/_index) to ensure the creation of high quality code.
```
Comment (nderjung) [inline content/en/community/_index.md:16]:
```diff
- [guidelines for development and maintenance](docs/contributing/_index)
+ [guidelines for development and maintenance](/docs/contributing)
```

File: content/en/community/_index.md (hunk 1 above)
Comment (razvand) [reply content/en/community/_index.md]: solved

File: content/en/community/_index.md (hunk 2 above)
Comment (razvand) [reply content/en/community/_index.md]: solved

**Review by nderjung — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**